app/api/wallet/network.ts

```diff
@@ -24,3 +24,19 @@ export function getBip32Option (envNetwork: EnvironmentNetwork): Bip32Options {
     wif: network.wifPrefix
   }
 }
+
+export function getTxURLByNetwork (network: EnvironmentNetwork, txId: string): string {
+  switch (network) {
+    case EnvironmentNetwork.MainNet:
+      return 'https://mainnet.defichain.io/#/DFI/mainnet/tx/' + txId
+
+    case EnvironmentNetwork.TestNet:
+      return 'https://testnet.defichain.io/#/DFI/testnet/home/tx/' + txId
+
+    case EnvironmentNetwork.LocalPlayground:
+      return 'http://localhost:19553/v0.7/regtest/transactions/' + txId
```

Comment:
this one will fail because it's not always v0.7

Reply:
any endpoint that we can call to retrieve the correct url path?